accepted/0000-caasp-integration.md

```diff
@@ -0,0 +1,375 @@
+
+##  Future work
+
+### CVE Auditing
+
+A useful addition would be to have SUSE Manager as an admission controller for rejecting pod creation whenever an image used is vulnerable.
+
+### Deploy OpenStack Cloud
+
+If OpenStack Cloud will be deployed using AirShip (still under discussion), the approach described in this RFC can be further reused for deploying and managing an OpenStack Cloud, given that `skuba` is similar to `airshipctl`:
+
+- provision nodes to be part of the cloud
+- tag nodes and bootstrap them to be part of the cloud
+- manage the lifecycle of the cloud
+
+### Run other products in the cluster
+
+It would also be possible, after that the cluster has been deployed, to deploy other SUSE products on top of the cluster (e.g. SUSE OpenStack Cloud, SUSE Enterprise storage): 
+
+- CaaS Platform deploys a SUSE Application Lifecycle operator on top of the cluster with a set of custom resources (CRDs)
+- SUSE Manager can then deploy SOC, SES or any other combination of products on top of the cluster and manage the lifecycle of the deployed products by simply having access to the `kubectl` configuration file
+
+In this case, SUSE Manager would be the central point of deployment for other products based on CaaS Platform and show the health of every cluster installed, while offering the possibility of managing and upgrading the cluster.
+
+### Kubernetes dashboard
+
+In the future developments, SUSE Manager can also embed the Kubernetes dashboard to help the user in deploy applications on top of CaaS Platform.
+The alternative of offering to configure application deployments from SUSE Manager would be seen as a second-choice solution compared to Kubernetes dashboard.
+
+### Re-use of deployment structure
+
+A small part of the deployment of the CaaS Platform can be reused for other Kubernetes distributions (e.g. OpenShift); deploying on bare metal requires machines configured with very specific requirements and that can be solved with AutoYaST (or Kickstart in the Red Hat case).
+
+Every distribution uses then its deployment helper (OpenShift uses `openshift-install`) to bootstrap the cluster. In this regard, `skuba` is only targeting SUSE CaaS Platform and cannot be reused for bootstrapping and managing a cluster that is not CaaS Platform based.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+What if the customer wants to deploy another Kubernetes distribution?
+If we are using `skuba`, we only target CaaS Platform deployments.
+
+# Alternatives
+[alternatives]: #alternatives
+
+- Managing the cluster with `kubectl` and importing its `kubeconfig` without using `skuba`: limited functionality, every Kubernetes distribution must be treated according to its specification and provide an actuator for the `cluster-api`, lots of duplicate work.
```

Comment:
As with the previous comment, `kubectl` can be implemented in addition to offer a second, more generic "Kubernetes" type of cluster.